package.json

```diff
@@ -26,6 +26,7 @@
     "mini-css-extract-plugin": "^2.4.5",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
+    "react-icons": "^4.4.0",
```

Comment:
How much does this add-up to the overall payload? Is it only the icons that are being requested or is the whole set downloaded to the client?

Reply:
it only adds the icons that requested to the payload